be/src/olap/null_predicate.h

```diff
@@ -54,10 +54,13 @@ class NullPredicate : public ColumnPredicate {
     }
 
     bool evaluate_del(const std::pair<WrapperField*, WrapperField*>& statistic) const override {
+        // evaluate_del only use for delete condition to filter page, need use delete condition origin value,
+        // when opposite==true, origin value 'is null'->'is not null' and 'is not null'->'is null',
```

Comment:
no need to check if opposite==false ?

Reply:
only deletehandler set opposite=true, when opposite==false, 'is null' is 'is null' and 'is not null' is 'is not null'.
and evaluate_del only use for delete condition to filter page.